installer/bundle/create_bundle.sh

```diff
@@ -40,6 +40,7 @@ usage()
 DIRECTORY=$1
 INTERMEDIATE=$2
 TAR_FILE=$3
+INSTALL_TYPE=$4
```

Comment:
This is perfect, but you should change the usage() output, above, to document what this means and how it is used.